code/game/objects/items/storage/firstaid.dm

```diff
@@ -14,11 +14,20 @@
 	throw_range = 8
 	use_sound = "toolbox"
 	matter = list("plastic" = 2000)
-	cant_hold = list(
-		/obj/item/ammo_magazine,
-		/obj/item/explosive/grenade,
-		/obj/item/tool,
-	) //to prevent powergaming.
+	can_hold = list(
```

Comment:
Should be able to hold:
- bloodbags, considering they're a medical item
- MREs, considering how often i need to tell people "eat food" as a medic

Reply:
can't hold surgical cases anymore either

Reply:
Not interested in MREs, first aid kits should not be used as a general equipment kit. Bloodbags I did miss.

Reply:
Stasis bags are a purely medical item too.